docs/release-notes/8.1.asciidoc

```diff
@@ -1,6 +1,18 @@
 [[release-notes-header-8.1.0]]
 == 8.1
 
+[discrete]
+[[release-notes-8.1.1]]
+=== 8.1.1
+
+[discrete]
+[[bug-fixes-8.1.1]]
+==== Bug fixes and enhancements
+* Fixes a bug in the Edit Rule Exception flyout that was truncating error messages ({pull}126875[#126875]).
+* Adds a notification to the **Exception lists** page that informs users whether they are lacking certain role privileges ({pull}126874[#126874]).
+* Turns the **Upload value lists** option off on the **Rules** page if users have `Read` Security privileges only ({pull}126829[#126829]).
```

Comment:
Very minor optional edit to avoid awkward "off on" construction:
```suggestion
* Turns off the **Upload value lists** option on the **Rules** page if users have `Read` Security privileges only ({pull}126829[#126829]).
```